src/platform/assets/components/AssetCard.vue

```diff
@@ -0,0 +1,91 @@
+<template>
+  <div
+    data-component-id="AssetCard"
+    :data-asset-id="asset.id"
+    role="button"
+    tabindex="0"
+    :aria-label="`Select asset ${asset.name}`"
+    :class="
+      cn(
+        'rounded-xl overflow-hidden cursor-pointer transition-all duration-200 min-w-60 max-w-64',
+        'bg-ivory-100 border border-gray-300',
+        'dark-theme:bg-charcoal-400 dark-theme:border-charcoal-600',
+        'hover:transform hover:-translate-y-0.5 hover:shadow-[0_8px_24px_rgba(0,0,0,0.1)] hover:border-gray-400',
+        'dark-theme:hover:shadow-[0_8px_24px_rgba(0,0,0,0.3)] dark-theme:hover:border-charcoal-700',
+        'focus:outline-none focus:ring-2 focus:ring-blue-500 dark-theme:focus:ring-blue-400'
+      )
+    "
+    @click="$emit('select', asset)"
+    @keydown.enter="$emit('select', asset)"
+    @keydown.space.prevent="$emit('select', asset)"
```

Comment:
If these are meant to stay in sync, I'd probably make a named function and put it in all 3 places.